src/algorithms/algorithm-v1.ts

```diff
@@ -27,4 +27,4 @@ class TryAgainAlgorithm implements DrawAlgorithm {
         return Math.floor(Math.random() * this.numbers.length);
     }
 }
-export { TryAgainAlgorithm };
\ No newline at end of file
+export { TryAgainAlgorithm };
```

Comment:
What has changed here? It looks the same to me, no new line? GitHub thinks there is a change? 🤔

Reply:
No need to use method getTime(). Date.now() between and after the action returns exact milliseconds from the epoch. If you capture time before and after the event and deduct start from end, the exact difference is the time the event has taken.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/now

Reply:
Both methods are correct but there is nuance and documentation points to use Date.now() instead of the get.Time() method to prevent unnecessary instantiating Date objects.
For reference:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTime#measuring_execution_time

Reply:
I don't agree. If you want to just use longs to capture time, the whole point of having a dedicated object to do this operation becomes meaningless. IMHO, having the object and abstracting this from the client class makes more sense, and you can reuse the logic anywhere you need a `StopWatch`.
We are trying to have a nice OOP here. 😄

Reply:
This is a simple example. But the reason I created this object was to show you this concept.
Think about something complex. Reusing logic is better than dealing with global functions.

Reply:
Ok, I will fix that :)